src/OMSimulatorPython/OMSimulator.py.in

```diff
@@ -1,13 +1,21 @@
-from ctypes import cdll
-import ctypes, sys, os
+from ctypes import CDLL
+import ctypes, sys, os, platform
 
 class OMSimulator:
   def __init__(self, omslib=None, temp_directory=None):
     if omslib is None:
       dirname = os.path.dirname(__file__)
-      self.obj=cdll.LoadLibrary(os.path.join(dirname, "@OMSIMULATORLIB_DIR_STRING@", "@OMSIMULATORLIB_STRING@"))
+      ## use CDLL instances for proper loading in Python versions > 3.8 as winmode=0 parameter is needed for proper loading in windows
+      if platform.python_version() >= "3.8":
+        self.obj=CDLL(os.path.join(dirname, "@OMSIMULATORLIB_DIR_STRING@", "@OMSIMULATORLIB_STRING@"), winmode=0)
+      else:
+        self.obj=CDLL(os.path.join(dirname, "@OMSIMULATORLIB_DIR_STRING@", "@OMSIMULATORLIB_STRING@"))
+      self.obj=CDLL(os.path.join(dirname, "@OMSIMULATORLIB_DIR_STRING@", "@OMSIMULATORLIB_STRING@"), winmode=0)
```

Comment:
I guess this line should be removed.

Reply:
removed https://github.com/OpenModelica/OMSimulator/pull/793/commits/76e3d3e78f6b902f35b31a7708d306093b1587e5